modular_skyrat/modules/biohazard_blob/code/biohazard_blob_disease.dm

```diff
@@ -1,46 +1,52 @@
-/datum/disease/cordyceps
-	form = "Disease"
-	name = "Cordyceps omniteralis"
-	max_stages = 5
+/datum/disease/cryptococcus
+	name = "Cryptococcal meningitis"
+	max_stages = 4
+	stage_prob = 1.75
 	spread_text = "Airborne"
-	cure_text = "Neurine & Modafinil"
-	cures = list(/datum/reagent/medicine/neurine, /datum/reagent/medicine/modafinil)
-	agent = "Fungal Cordycep bacillus"
+	spreading_modifier = 0.75
+	cure_text = "Haloperidol"
+	cures = list(/datum/reagent/medicine/haloperidol)
+	agent = "Cryptococcus gattii fungus"
 	viable_mobtypes = list(/mob/living/carbon/human)
-	cure_chance = 20
-	desc = "Fungal virus that attacks patient's muscles and brain in an attempt to hijack them. Causes fever, headaches, muscle spasms, and fatigue."
+	cure_chance = 25
+	desc = "Fungal infection that attacks patient's muscles and brain in an attempt to hijack them. Causes fever, headaches, muscle spasms, and fatigue."
 	severity = DISEASE_SEVERITY_BIOHAZARD
 	bypasses_immunity = TRUE
 
-/datum/disease/cordyceps/stage_act(delta_time, times_fired)
+/datum/disease/cryptococcus/stage_act(delta_time, times_fired)
 	. = ..()
 	if(!.)
 		return
 
 	switch(stage)
-		if(2)
+		if(1)
 			if(DT_PROB(2, delta_time))
+				to_chat(affected_mob, span_danger("Your brain feels fuzzy. That's not right, you should get closer to medbay..."))
+			if(DT_PROB(5, delta_time))
+				to_chat(affected_mob, span_danger("Your head hurts. Were the ceiling tiles always moving like that? You should get closer to medbay..."))
+		if(2)
+			if(DT_PROB(5, delta_time))
 				affected_mob.emote("twitch")
-				to_chat(affected_mob, span_danger("You twitch."))
+				to_chat(affected_mob, span_danger("You twitch involuntarily. That's not right, you should get closer to medbay..."))
 			if(DT_PROB(2, delta_time))
-				to_chat(affected_mob, span_danger("You feel tired."))
-			if(DT_PROB(5, delta_time))
-				to_chat(affected_mob, span_danger("Your head hurts."))
-		if(3,4)
+				to_chat(affected_mob, span_danger("You sniff, smelling green slime. Does green have a smell? You should get closer to medbay..."))
+			if(DT_PROB(2, delta_time))
+				to_chat(affected_mob, span_danger("Your head hurts. Were the ceiling tiles always moving like that? You should get closer to medbay..."))
```

Comment:
I'm fine with these except I don't think they should mention getting closer to Medbay. Everything else is good.

Reply:
Done